src/routes/+page.svelte

```diff
@@ -33,17 +50,21 @@
                     <AudioIcon.Mute _class="dy-swap-off fill-black-100" />
                 </label>
             </button>
-            <!-- Search Button -->
-            <a href="/search" class="dy-btn dy-btn-ghost dy-btn-circle">
-                <SearchIcon />
-            </a>
+            {#if showSearch}
+                <!-- Search Button -->
+                <a href="/search" class="dy-btn dy-btn-ghost dy-btn-circle">
+                    <SearchIcon />
+                </a>
+            {/if}
             <!-- Text Appearance Options Menu -->
-            <Dropdown>
-                <svelte:fragment slot="label">
-                    <TextAppearanceIcon />
-                </svelte:fragment>
-                <!-- TODO: implement text appearance options -->
-            </Dropdown>
+            {#if showTextAppearnce}
```

Comment:
Please create a TextAppearanceSelector like the ChapterSelector and BookSelector and put all of the TextAppearance code (countThemes* and showTextAppearance) in that component.